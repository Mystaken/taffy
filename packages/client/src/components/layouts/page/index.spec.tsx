import React from 'react';
import { create } from 'react-test-renderer';
import { Page } from '.';

describe('<Page />', () => {
  it('has rendered correctly', () => {
    const component = create(<Page />);
    const tree = component.toJSON()!;
    expect(tree).toMatchSnapshot();
  });
  it('has rendered correctly with a title', () => {
    const component = create(<Page title="test" />);
    const tree = component.toJSON()!;
    expect(tree).toMatchSnapshot();
  });
});
