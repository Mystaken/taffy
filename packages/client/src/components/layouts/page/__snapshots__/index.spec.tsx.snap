// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Page /> has rendered correctly 1`] = `null`;

exports[`<Page /> has rendered correctly with a title 1`] = `null`;
